src/pages/MyPageEdit.tsx

```
@@ -4,58 +4,62 @@ import styled from 'styled-components';
+  &::before {
+    content: '';
+    position: absolute;
+    bottom: -12px;
+    left: 0;
+    height: 1px;
+    width: 100%;
+    background-color: ${colors.semantic.line};
+  }
 `;
 
 const Error = styled.div`
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)